tests/testthat/apps/app/tests/shinytest/mytest.R

```diff
@@ -1,4 +1,4 @@
-app <- ShinyDriver$new("../../")
-app$snapshotInit("mytest")
-
-app$snapshot()
+app <- ShinyDriver$new("../../", loadTimeout = 10000)
```

Comment:
I think this will also fix the timeout error that I'm experiencing on my Windows laptop. Thnx!